EDMC.py

```diff
@@ -62,21 +63,41 @@ def versioncmp(versionstring):
     args = parser.parse_args()
 
     if args.version:
-        latest = ''
+        newversion = None
         try:
-            # Copied from update.py - probably should refactor
             r = requests.get(update_feed, timeout = 10)
-            feed = ElementTree.fromstring(r.text)
-            items = dict([(item.find('enclosure').attrib.get('{http://www.andymatuschak.org/xml-namespaces/sparkle}version'),
-                           item.find('title').text) for item in feed.findall('channel/item')])
-            lastversion = sorted(items, key=versioncmp)[-1]
-            if versioncmp(lastversion) > versioncmp(appversion):
-                latest = items[lastversion]
-        except Exception as e:
-            sys.stderr.write('Exception in version check: {}'.format(str(e)))
-            #pass	# Quietly suppress timeouts etc.
-        if latest:
-            print('{CURRENT} ({UPDATE} is available)'.format(CURRENT=appversion, UPDATE=latest))
+        except requests.RequestException as ex:
+            sys.stderr.write('Error retrieving update_feed file: {}\n'.format(str(ex)))
+        else:
+            try:
+                feed = ElementTree.fromstring(r.text)
+            except SyntaxError as ex:
+                sys.stderr.write('Syntax error in update_feed file: {}\n'.format(str(ex)))
```

Comment:
```suggestion
                print('Syntax error in update_feed file: {}'.format(str(ex)), file=sys.stderr)
```

Print makes it cleaner as to what you're doing